public/app/overrides/components/Toolbar.tsx

```diff
@@ -14,7 +14,7 @@ import Button from '@webapp/ui/Button';
 import LoadingSpinner from '@webapp/ui/LoadingSpinner';
 import DateRangePicker from '@webapp/components/DateRangePicker';
 import RefreshButton from '@webapp/components/RefreshButton';
-import AppSelector from '@phlare/components/AppSelector/AppSelector';
+import { AppSelector } from '@phlare/components/AppSelector/AppSelector';
```

Comment:
It's funny because both are technically valid.

```
import { AppSelector } from '@phlare/components/AppSelector/AppSelector';
```

specifies only importing `AppSelector`, but since `AppSelector` is declared as the default export in that module, what was originally there is functionally equivalent.